playwright.config.ts

```diff
@@ -20,9 +20,12 @@ export default defineConfig({
         ['./ci-reporter'],
       ]
     : 'list',
-  timeout: process.env.CI ? 60000 : 30000,
-  expect: { timeout: process.env.CI ? 30000 : 20000 },
+  globalTimeout: 29.5 * 60 * 1000, // Set because of codebuild, we want PW to timeout before CB to get the results.
+  timeout: 10 * 60 * 1000,
+  expect: { timeout: process.env.CI ? 30_000 : 20_000 },
   use: {
+    actionTimeout: 30_000,
+    navigationTimeout: 30_000,
```

Comment:
5 different timeouts :D 

I would appreciate comment in the code on what is `timeout`, how does `expect` differ from `actionTimeout`.

And maybe let's just define all of them as 30 000? I think the `CI` check might not be necessary - slow computer or slow network at home might be worse that CI :D

Reply:
That global one is just because of codebuild and results, that's easy. And if want more granular timeouts, we have to set all of those that are more down the line from the test one (i.e. that's just the `timeout`), because defaults for those are 0 which means unlimited. 😄 
Yeah the CI might not be needed in the expect one 😅 .

Reply:
We are going to have a lot of people working in this repo, I'd suggest just changing all of these values to their int equivalents to simplify this. Maybe simply have the number followed by the comment of the time it equates to.

Simplicity is king. 

number // x minutes